webapps/src/main/webapp/vue-app/taskCommentsDrawer/components/TaskCommentEditor.vue

```diff
@@ -31,6 +31,7 @@
         :placeholder="placeholder"
         :object-id="newCommentId"
         :tag-enabled="false"
+        :suggester-space-url="getSpaceUrlFromProjectParticipator()"
```

Comment:
Please add it as computed value instead of a method

```suggestion
        :suggester-space-url="taskSpaceUrl"
```